modules/beam/src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala

```diff
@@ -197,4 +200,15 @@ object config {
   def parseLabels(input: String): Either[String, Map[String, String]] =
     decode[Map[String, String]](input)
       .leftMap(_ => s"Invalid `labels` format, expected json object, received: $input")
+
+  /** Create a java.net.URI from string with Sentry DSN. */
+  private def parseSentryDsn(maybeDSN: Option[String]): Either[String, Option[URI]] =
+    maybeDSN match {
+      case Some(uri) =>
+        Either
+          .catchNonFatal(URI.create(uri))
```

Comment:
It in fact fails on very few sets invalid URIs, unfortunately, like `/` is valid. I think eventually we want to make this check more restrictive.

Reply:
Indeed we could. Should I at least check that it starts with `http://` or `https://` ?

Reply:
I don't know. Probably. If we're sure Sentry supports HTTP(S) only.

Reply:
Seems that this is a REST API : https://docs.sentry.io/api/requests/
I'm adding this additional check